pkg/flare/archive_dca.go

```diff
@@ -176,3 +180,95 @@ func getDCAWorkloadList() ([]byte, error) {
 
 	return getWorkloadList(fmt.Sprintf("https://%v:%v/workload-list?verbose=true", ipcAddress, config.Datadog.GetInt("cluster_agent.cmd_port")))
 }
+
+// getAgentDaemonSet retrieves the DaemonSet manifest of the Agent
+func getAgentDaemonSet() ([]byte, error) {
```

Comment:
IMO this function can be also move to the `manifests.go` like `getClusterAgentDeployment` and `getHelmValues` so only this 3 functions needs to be present in `manifests_nocompile.go`